src/apps/lwaftr/fragmentv4_test.lua

```diff
@@ -90,11 +119,21 @@ local function check_packet_fragment(orig_pkt, frag_pkt, is_last_fragment)
    assert(orig_hdr:dst_eq(frag_hdr:dst()))
    assert(orig_hdr:type() == frag_hdr:type())
 
+   local eth_size = eth_proto:sizeof()
+   if orig_hdr:type() == 0x8100 then
+      eth_size = eth_size + 4
+      -- Check extra 4 bytes: VLAN tags and protocol type
```

Comment:
To me this would be more clear if we checked that the VLAN ID and the nested ethertype instead of doing this bytewise comparison, but ymmv...